shared/schemas/query.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "object",
+  "properties": {
+    "body": { "type": "object" },
+    "filter_path": { "type": "string" },
+    "from": { "type": "number" },
```

Comment:
this (and "size") can be dropped from the top level since it's supported by the body format

Reply:
I tried to add size and from to query, it doesn't work

![Screenshot from 2021-03-05 10-07-13](https://user-images.githubusercontent.com/533229/110079842-aa44c000-7d9a-11eb-8fe5-5301f44c86ef.png)